maps/CdaToBundle.4.map

```diff
@@ -1047,6 +1047,99 @@ group CdaPatientRoleToFhirPatient(source cda_patientRole : PatientRole, target f
   } "CdaPatientToFhirPatient";
 }
 
+group CdaPatientNameToFhirPatientName(source cda_patient_name : PN, target fhir_patient : Patient){
+  
+  // For each family element, a HumanName is created.
+  // For family elements with qualifier="BR" or "AD", only the given elements are included.
+  cda_patient_name.item as cda_name_item then {
```

Comment:
We need this line only once, I think. So the other parts ("AD" and "not AD and not BR") could be part of this one.